models/player.go

```diff
@@ -1,6 +1,146 @@
 package models
 
+import (
+	"fmt"
+	"github.com/TeamPlayTF/Server/database"
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
 type Player struct {
-	steamid string //Players steam ID
-	name    string //Player name
+	Id      bson.ObjectId `bson:"_id,omitempty"        json:"id"`      // MongoDB ID
```

Comment:
the getters have useful (for lazy ppl) functions, like automatically convert into string.
the setters will need length checking, steamid should have fixed length and name should have maxLength.